newshomepages/sources/bundles.csv

```diff
@@ -9,3 +9,4 @@ bay-area,California Bay Area,America/Los_Angeles,San Francisco
 iowa,Iowa,America/Chicago,Iowa City
 new-mexico,New Mexico,America/Denver,Albuquerque
 us-right-wing,U.S. right wing,America/New_York,Washington
+north-carolina,North Carolina,America/New_York,North Carolina
```

Comment:
The final location column is intended to be a representative city or landmark. The name of the bundle is already North Carolina. By having a more specific places in the location field we're able to write tweets like:

> U.S. national television homepages at 2:10 PM in New York

What places would you pick to represent North Carolina? I generally think a city that is a media hub works well.